ipsec_offload_plugin/ipsec_offload/ipsec_offload.c

```diff
@@ -70,6 +74,7 @@ enum ipsec_status ipsec_rx_sa_classification_table(enum ipsec_table_op table_op,
 						char src_ip_addr[16],
 						uint32_t spi,
 						uint32_t offloadid);
+#ifdef STANDALONE_IPSEC_RECIPE
```

Comment:
I believe we can't decide if its standalone/merged IPSec recipe this way. The standalone and merged should live together. Its the SS configuration which decides which tables need to be configure. We have to refactor the code based on some configurations from SS. #ifdef are not recommended.

The reason: This recipe is platform independent. Its doesn't need to know which recipe mev/non-mev etc.

Reply:
ifdefs are to remove additional parameters which are not required for new tables in v3. I can keep the parameters and not use them if you prefer that approach

Reply:
If you think keeping table names different altogether to avoid ifdefs is a better approach, please let me know and I can make necessary adjustments